cmd/fleetctl/flags.go

```diff
@@ -57,3 +57,17 @@ func stdoutFlag() cli.Flag {
 func getStdout(c *cli.Context) bool {
 	return c.Bool(stdoutFlagName)
 }
+
+func byHostIdentifier() cli.Flag {
+	return &cli.StringFlag{
+		Name:  "host",
+		Usage: "The host identifier to operate on",
```

Comment:
Jacob is adding this everywhere:
https://github.com/fleetdm/fleet/pull/20220/files#diff-f3c33e613be73d3a842f768be512c63f8160cc932110c48aea7b425b2c03f12dR209

Maybe we should add it here: `"The host, specified by hostname, UUID, or serial number"`